docker-compose.yml

```diff
@@ -4,16 +4,9 @@ services:
     build: 
         context: .
         dockerfile: src/Miki/Dockerfile
+        args:
+          private_nuget_pat: ${PRIVATE_NUGET_PAT}
```

Comment:
We will need to modify the pipeline so that this variable gets passed, PAT token can be created in DevOps (https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=preview-page)